docs/reference/inference/inference-shared.asciidoc

```diff
@@ -63,4 +63,44 @@ Specifies the chunking strategy.
 It could be either `sentence` or `word`.
 end::chunking-settings-strategy[]
 
+tag::unified-schema-content-with-examples[]
+.Examples
+[%collapsible%closed]
+======
+String example
+[source,json]
+------------------------------------------------------------
+{
+    "content": "Some string"
+}
+------------------------------------------------------------
+// NOTCONSOLE
+
+Object example
+[source,json]
+------------------------------------------------------------
+{
+    "content": [
+        {
+            "text": "Some text",
+            "type": "text"
+        }
+    ]
+}
+------------------------------------------------------------
+// NOTCONSOLE
+======
 
+String representation:::
+(Required, string)
+The text content.
++
+Object representation:::
+`text`::::
```

Comment:
We get lucky that each of the messages has `content::` so the colons work out here to nest it correctly.